clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/UbamTests.scala

```diff
@@ -313,6 +313,61 @@ trait UbamTests { self: BaseIntegrationSpec =>
     }
   }
 
+  it should "handle querying ubams by aggregated-by" in {
```

Comment:
Is there something unique about the "aggregated-by" field that necessitates testing that we can query by it specifically?

Reply:
Yeah. Its for Ops to be able to query all ubams in a specific aggregation project. Without this field, Ops double counts some ubams because they might be in the same Squid project but not  the same research project.